Rakefile

```diff
@@ -227,100 +225,146 @@ namespace :jsondoc do
     mkdir_p gh_pages + "json/stackdriver/master/google/cloud/"
   end
 
-  desc "Copies all gems jsondoc to gh-pages repo in temp dir."
-  task :copy_all, [:tag] => [:init] do |t, args|
-    tag = args[:tag]
-    fail "Missing required parameter 'tag'." if tag.nil?
-    gh_pages = Pathname.new(Dir.home) + "tmp/#{tag}-gh-pages"
+  desc "Copies a gem's jsondoc to gh-pages repo in temp dir."
+  task :copy, [:gem, :version, :gh_pages_dir] do |t, args|
+    gem, version, gh_pages_dir = extract_args args, :gem, :version, :gh_pages_dir
+    gh_pages = gh_pages_path gh_pages_dir
 
-    header "Copying all jsondoc for branch/tag '#{tag}' to '#{gh_pages}'"
-    gems.each do |gem|
-      unless Dir.exist? gh_pages + "json/#{gem}"
-        mkdir_p gh_pages + "json/#{gem}", verbose: true
-      end
-      gem_version = tag.include?("/") ? tag.split("/").last : tag
-      rm_rf gh_pages + "json/#{gem}/#{gem_version}", verbose: true
-      mkdir_p gh_pages + "json/#{gem}/"
-      cp_r "#{gem}/jsondoc", gh_pages + "json/#{gem}/#{gem_version}", verbose: true
+    header "Copying #{gem} jsondoc for '#{gem}/#{version}' to '#{gh_pages}'"
+
+    unless Dir.exist? gh_pages + "json/#{gem}"
+      mkdir_p gh_pages + "json/#{gem}", verbose: true
     end
+    rm_rf gh_pages + "json/#{gem}/#{version}", verbose: true
+    mkdir_p gh_pages + "json/#{gem}/"
```

Comment:
Yes, sure looks like it! I decided to wrap in that conditional on 235 and I forgot to delete the original I think. Good catch.